contracts/astroport-liquidity-helper/src/contract.rs

```diff
@@ -190,93 +186,51 @@ pub fn execute_balancing_provide_liquidity(
                 }
             }
 
-            let mut response = Response::new();
+            let mut swap_res = Response::new();
```

Comment:
I'd prefer remove this and do
```
if ... {
pool.swap(...)
} else {
Response::new()
}
```
no need for `mut`

Reply:
true

Reply:
fixed